src/Devices/Ethernet_TC.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Ethernet_TC.h"
+using namespace std;
+
+TC_ethernet *TC_ethernet::_instance = nullptr;
```

Comment:
Could you change the name of the class to match the filename? Please use `Ethernet_TC`.